docs/config/README.md

```diff
@@ -3756,4 +3761,10 @@ format = " transcending [$output]($style)"
 command = "time /T"
 extensions = ["pst"] # filters *.pst files
 shell = ["pwsh.exe", "-NoProfile", "-Command", "-"]
+
+[custom.time-as-arg]
+command = "time /T"
+extensions = ["pst"] # filters *.pst files
```

Comment:
Should we change the examples to use the new `detect_*` rather than the old versions?

Reply:
Sounds good. Done!